py/htm/encoders/grid_cell_encoder.py

```diff
@@ -50,8 +51,8 @@ def __init__(self,
         encoder uses.  This encoder produces deterministic output.  The seed
         zero is special, seed zero is replaced with a truly random seed.
         """
-        self.size       = size
-        self.dimensions = (size,)
+        super().__init__((size,))
```

Comment:
:+1: this is a correct use of explicitely calling the super's methods. (note: unlike in c++, you dont need to call the super.init as the first statement